main.cpp

```diff
@@ -13,19 +13,21 @@ wgpu::Device device;
 wgpu::RenderPipeline pipeline;
 
 wgpu::Surface surface;
+wgpu::TextureFormat format;
 const uint32_t kWidth = 512;
 const uint32_t kHeight = 512;
 
 void ConfigureSurface() {
   wgpu::SurfaceCapabilities capabilities;
   surface.GetCapabilities(adapter, &capabilities);
+  format = capabilities.formats[0];
 
   wgpu::SurfaceConfiguration config{
       .device = device,
+      .format = format,
+      .alphaMode = capabilities.alphaModes[0],
```

Comment:
Oops, I had a draft suggestion that I never submitted.

```suggestion
      .alphaMode = wgpu::CompositeAlphaMode::Auto,
```